### ISSUE TITLE
Contribute Java Programs

### ISSUE THREAD

**Issue by codec-akash** [codec-akash/Java-Programs#1]

**Comment by charchitmangal** [comment]
Hello! I would Like to Work on this Issue

**Comment by sandhukaran28** [comment]
Hey, I would like to contribute.

**Comment by codec-akash** [comment]
> Hello! I would Like to Work on this Issue

hello, feel free to contribute but just contribute legitimate and working java programs.

**Comment by codec-akash** [comment]
> Hey, I would like to contribute.

hello, feel free to contribute but just contribute legitimate and working java programs.

**Comment by shashwat-agarwal** [comment]
Hey , I would like to contribute .

**Comment by codec-akash** [comment]
> Hey , I would like to contribute .

Yup, Please feel free to contribute you can add some of the java programs which are not posted in the repository.

**Comment by sahanihit** [comment]
This Repository has been marked as "Invalid" by Hacktoberfest.
Please close down this issue.

**Comment by twentyonedot** [comment]
@sahanihit are you sure, cuz my pr has been accepted as valid one.

**Comment by twentyonedot** [comment]
@codec-akash  Can you please tell that this repository is still valid for hacktoberfest?

**Comment by codec-akash** [comment]
Yes after the new norms from the dev fest team. I guess(not sure though) that this repository is not applicable for hacktoberfest.
Still we should contribute to open source even if we don't get anything in return. It might help others.

**Comment by twentyonedot** [comment]
@codec-akash Yeah I agree. I will definitely contribute to this repo.

**Comment by devmalik7** [comment]
I appreciate it

**Comment by codec-akash** [comment]
> I appreciate it

Thanks!! this would help people learn and spread knowledge of the beautiful language Java.
